src/main/kotlin/com/openlattice/launchpad/configuration/Integration.kt

```diff
@@ -31,8 +31,10 @@ import java.util.*
 const val SOURCE = "source"
 const val DESTINATION = "destination"
 
+
 data class Integration(
-        @JsonProperty(DESCRIPTION) val description: Optional<String>,
+        @JsonProperty(DESCRIPTION) val description : String = "",
```

Comment:
is `DESCRIPTION` defined somewhere? Confused about where this constant is coming from...

Reply:
Comes from `IntegrationConfiguration.kt`